deployment/network-operator/values.yaml

```diff
@@ -18,6 +18,7 @@
 
 nfd:
   enabled: true
+  nodefeaturerules: true
```

Comment:
nit:

deployNodeFeatureRules ?

Reply:
need to update documentation under deployment

Reply:
Done